pkg/cmd/server/cacheconfig.go

```diff
@@ -58,8 +58,8 @@ func (cc *CacheConfig) Complete() (*cache.Config, error) {
 // RegisterCacheConfigFlags registers flags for a ristretto-based cache.
 func RegisterCacheConfigFlags(flags *pflag.FlagSet, config *CacheConfig, flagPrefix string) {
 	flagPrefix = stringz.DefaultEmpty(flagPrefix, "cache")
-	flags.StringVar(&config.MaxCost, flagPrefix+"-max-cost", defaultMaxCost, "the maximum cost to be stored in the cache, in bytes")
-	flags.Int64Var(&config.NumCounters, flagPrefix+"-num-counters", defaultNumCounters, "the number of keys to track")
+	flags.Int64Var(&config.MaxEntries, flagPrefix+"-max-entries", defaultMaxEntries, "maximum number of cache entries")
```

Comment:
Maybe we should have the old flag be a hidden and if specified, have it warn or fail?